nx-libs.spec

```diff
@@ -357,6 +357,10 @@ find -name LICENSE | xargs sed -i \
   -e 's/MA  02111-1307/MA  02110-1301/'
 # Fix source permissions
 find -type f -name '*.[hc]' | xargs chmod -x
+# RHEL6 and lower do not know -Wpedantic
+%if 0%{rhel} < 7
```

Comment:
Same problem as in the other commit (missing `?` char and the other check.)

While this may fix builds for EL6-, it would be good to also fix *SUSE builds if we're at it.

Looks like `-Wpedantic` is supported from GCC 4.8 onwards, and...

SLE 12 has GCC 4.8, SLE 11 has 4.3.

OpenSUSE 13.* has GCC 4.8, so that's fine. 12.* only has GCC 4.7, too old.

So how about:

```spec
%if ( 0%{?rhel} && 0%{?rhel} < 7 ) || ( 0%{?suse_version} && 0%{?suse_version} < 1310 )
```

?